build-system/tsconfig.json

```diff
@@ -27,15 +27,18 @@
     "./common/**/*.ts",
     "./server/**/*.js",
     "./server/**/*.ts",
-    "./tasks/**/*.js",
-    "./tasks/**/*.ts"
+    "./tasks/e2e/*.js",
```

Comment:
same comment about combining *.ts and *.js lines.

Reply:
Pretty sure I wrote this prior to your other comment and didn't remember to make the change on this branch. Great suggestion as before.